data/governments.txt

```diff
@@ -497,6 +497,31 @@ government "Remnant"
 		"Alpha" -.05
 	"penalty for"
 		assist -0.25
+		disable 1
+		board 1
+		capture 10
+		destroy 10
+		scan 0.1
+	"provoked on scan"
+	"friendly hail" "remnant uncontacted"
+	"hostile hail" "remnant uncontacted hostile"
+
+government "Remnant (research)"
+	"display name" "Remnant"
+	swizzle 0
+	color .89 .38 .62
+	"crew defense" 2.2
+	
+	"player reputation" 1
+	"bribe" 0
+	"attitude toward"
+		"Indigenous Lifeform" 0.05
+		"Korath" -.05
+		"Alpha" -.05
+		"Remnant" 1
+	"penalty for"
+		assist -0.25
+		board 1
```

Comment:
Is it intentional that this government uses the default disable penalty? Are they suppose to use the same penalty as the normal government or 0?

Reply:
It was originally my intent to have the research government a dual purpose one for both "scan this Remnant" and "disable this Remnant" missions, so it was supposed to be 0. But I think I'll just make this Research govt just for scanning, and use a separate govt for disabling when it comes up. A little more duplication, but it supports all intended ship behaviors better. (like having the scannable Remnant properly supported by other Remnant if they get attacked)